config/webpack.js

```diff
@@ -12,6 +16,17 @@ module.exports = {
     },
     failOnError: true, // don't report error to grunt if webpack find errors
     watch: true, // use webpacks watcher
-    //keepalive: true, // don't finish the grunt task
   },
+  prod: {
+    devtool: 'source-map',
+    failOnError: true, // don't report error to grunt if webpack find errors
+    output: {
+      path: path.join(__dirname, '../public/dist/'),
+      filename: 'bundle.min.js',
```

Comment:
Yeah, that'll be something needing to be done when we generate the html files (e.g. for l10n too)